### DISCUSSION TITLE
Styling the RHF Controller

### DISCUSSION THREAD

**Discussion by KindArt** [react-hook-form/react-hook-form#1853] [Q&A] [answered]
Hi all,

I've noticed that whilst wrapping my component with the RHF controller, it renders my component within a div tag.

I'm using flexbox for some functionality and reading through the documentation, I can't see how I can pass css to the divs that the controller renders.

Any ideas?

**Comment by KindArt** [comment, marked as answer]
Resolved.

Within the `as` prop you can pass in children that you expect to be rendered.

**Reply by bluebill1049** [reply]
next major release will have render props, more freedom, and predictable result. `as` will be remained.
